index.js

```diff
@@ -158,29 +159,42 @@ const genRange = ({
       day,
       tasks: rt,
       hours: HOURS - unavails.length,
+      togglProject,
     }).forEach((v) => {
       console.log(Object.values(v).map(v => `"${v}"`).join(','))
     })
   }
 }
 
 if (require.main === module) {
-  // TODO: parameterize these
-  const after = '2020-10-01' // inclusive
-  const before = '2020-12-31' // inclusive
-  const peepoSheet = './employees_2020.csv'
+  const [after, before, peepoSheet, togglProject = 'Engineering'] = process.argv.slice(2)
```

Comment:
Could do some validation here:
```suggestion
  const [after, before, peepoSheet, togglProject = 'Engineering'] = process.argv.slice(2)
  if (!after || !before || !peepoSheet || !togglProject) {
    throw new Error('Usage: [...]')
  }
```

Reply:
i'd defer this to when i iterate to use a more robust (and with auto-help/usage hinting) library

> basic process.argv parsing, to be revised in the future with something like yargs or minimist

Reply:
oh, right, sorry I missed that.